### DISCUSSION TITLE
Shared library support and another suggestion 编译为动态库支持等三个改进细节

### DISCUSSION THREAD

**Discussion by taozuhong** [qax-os/excelize#2036] [General]
看了项目，觉得完成度比较高，星星和使用数量似乎也证明了这一点，非常了不起的成就哦；

有三个小细节看看能否支持以下：
1、增加编译为动态库的支持，编译为动态库可以让更多的语言使用，释放出更大的能量；
2、文档结构似乎有点散乱，可以稍加整理以下
3、增加单元测试用例覆盖度，对于保证质量大有裨益；

**Comment by xuri** [comment]
Thanks for your feedback. Link to issue #728. The [excelize-py](https://github.com/xuri/excelize-py) is a Python port of Go Excelize library based on [shared library](https://github.com/xuri/excelize-py/blob/e49e57c1fcebf45ad00d9615888fff8962d5ee79/.github/workflows/build.yml#L95). Contribution are welcome.

**Reply by taozuhong** [reply]
Got it, the MSYS MINGW package has submitted:
https://github.com/msys2/MINGW-packages/pull/22900

**Reply by xuri** [reply]
Great work, many thanks.
